package.json

```diff
@@ -13,27 +13,22 @@
     "Lorenzo Stoakes <lorenzo@resin.io>"
   ],
   "scripts": {
-    "lint": "resin-lint angular.coffee core.coffee request.coffee test",
+    "lint": "resin-lint test",
```

Comment:
We should probably also lint `.ts` files.

Reply:
I've added an issue for it in https://github.com/resin-io/pinejs-client-js/issues/57 - I want it to be coupled with auto-formatting (rather than having to manually add all the ; everywhere) so will look into it soon and add it everywhere